database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -274,21 +340,21 @@ class MetadataSlickDatabase(originalDatabaseConfig: Config)
                                                               (Option[WorkflowMetadataSummaryEntry], Seq[MetadataEntry]) => WorkflowMetadataSummaryEntry
                                                              )(implicit ec: ExecutionContext): DBIO[Unit] = {
 
-    val exactMatchMetadataKeys = Set(startMetadataKey, endMetadataKey, nameMetadataKey, statusMetadataKey, submissionMetadataKey, parentWorkflowIdKey, rootWorkflowIdKey)
-    val startsWithMetadataKeys = Set(labelMetadataKey)
-
-    val metadataEntries = rawMetadataEntries filter { entry =>
-      entry.callFullyQualifiedName.isEmpty && entry.jobIndex.isEmpty && entry.jobAttempt.isEmpty &&
-        (exactMatchMetadataKeys.contains(entry.metadataKey) || startsWithMetadataKeys.exists(entry.metadataKey.startsWith))
-    }
-    val metadataWithoutLabels = metadataEntries
-      .filterNot(_.metadataKey.contains(labelMetadataKey)) // Why are these "contains" while the filtering is "starts with"?
-      .groupBy(_.workflowExecutionUuid)
-    val customLabelEntries = metadataEntries.filter(_.metadataKey.contains(labelMetadataKey))
+    val summarizationPartitionedMetadata = partitionSummarizationMetadata(
```

Comment:
Since we now only writing summarizable metadata to the queue, do we need to partition it again on summarizer side here?

Reply:
When Cromwell with these changes starts up it won't know for sure that everything in the queue represents summarizable metadata.

Reply:
I thought we could have a liquibase-sql one-off migration to run through pre-existing rows in `SUMMARY_QUEUE_ENTRY` and delete those records which we are non-summarizable. This way we could simplify summarizer logic.

Reply:
Could be done as part of the `develop` edition, but not a show-stopper for the hotfix IMO

Reply:
Ooh I like that idea, I'll look at doing that for develop.